autopush/websocket.py

```diff
@@ -989,6 +989,7 @@ def finish_webpush_notifications(self, result):
         now = int(time.time())
         messages_sent = False
         message = self.db.message_table(self.ps.message_month)
+        message.touch_record(self.ps.uaid)
```

Comment:
Is this the best place for this? process_notifications/finish_webpush_notifications isn't called unoften, right?

Also calling it from here, it should really be called via a thread

Reply:
I suppose it makes sense if we ever expect that a websocket connection will live longer than the max expiration and only does checks. I'll grant that it's an edge case.